mesh/src/vertex/Cylindrical2dVertexMesh.cpp

```diff
@@ -364,7 +425,7 @@ void Cylindrical2dVertexMesh::CheckNodeLocation(Node<2>* pNode)
     {
         pNode->rGetModifiableLocation()[0] = x_location + mWidth;
     }
-    else if (x_location > mWidth)
+    else if (x_location >= mWidth)
```

Comment:
Why has this changed?

Reply:
The code is 

if (x_location < 0)
    {
        pNode->rGetModifiableLocation()[0] = x_location + mWidth;
    }
    else if (x_location >= mWidth)
    {
        pNode->rGetModifiableLocation()[0] = x_location - mWidth;
    }

So if the second one is < then modes  at mWidth aren't moved to 0 which isn't correct as as here 0=mWidth

Ive updated the hpp to have the correct closed/open interval [0,mWidth)